check-oracle-vars.yml

```diff
@@ -0,0 +1,21 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+ahf_extract_path: "/u01/AHF"
+orachk_script_dir: "/u01/orachk"
+orachk_rpt_filename: "/tmp/orachk-zipfile.txt"
+# Default is to NOT uninstall AHF
+uninstall_ahf: false
+# default is to NOT run orachk
+run_orachk: false
```

Comment:
Just had a thought here:  check-oracle says it's used to run orachk, but here the default is not to run it.  So I must be missing something here.

Reply:
We do not uninstall/install AHF if already installed - that's all it is.
If AHF has been uninstalled, then the install will take place.